app/Http/Controllers/GalaxyController.php

```diff
@@ -149,7 +155,12 @@ public function getGalaxyArray(int $galaxy, int $system, PlayerService $player,
                             'planetMovePossible' => true,
                             'moveAction' => 'prepareMove',
                             'title' => 'Relocate'
-                        ]
+                        ],
+                        $user_planet->getObjectAmount('colony_ship') > 0 ? [
+                            'missionType' => 7,
+                            'link' => "/fleet?galaxy={$galaxy}&system={$system}&position={$i}&type=1&mission=7",
+                            'description' => "Coloniser<br>Cette position est normalement occupée par des planètes équilibrées, avec assez de deutérium et suffisamment d`énergie solaire, qui offrent assez de place pour le développement.<br><div style='display: flex;align-items: center;'><img src='/cdn/img/galaxy/activity.gif' style=''/>Sans vaisseau de colonisation, aucune colonisation de planète n`est possible!</div>"
```

Comment:
Can you add the english translation here as default? I just checked the tooltip in the official game and the descriptions seem to be dynamic based on the position of the target planet. Examples:

<img width="444" alt="Screenshot 2024-06-04 at 22 04 39" src="https://github.com/lanedirt/OGameX/assets/6917405/76979649-0e58-4737-a4b6-fa9092a30a9d">

<img width="428" alt="Screenshot 2024-06-04 at 22 04 47" src="https://github.com/lanedirt/OGameX/assets/6917405/7d140e30-9c93-4a12-b6db-f316f7fd4e4c">

<img width="417" alt="Screenshot 2024-06-04 at 22 04 53" src="https://github.com/lanedirt/OGameX/assets/6917405/2ce401a9-1772-45f7-bb05-57303cbad528">

Reply:
Oh shit!
Yep i will add it, thanks :)